### DISCUSSION TITLE
Increase time precission in GPX files

### DISCUSSION THREAD

**Discussion by mikoreg** [osmandapp/OsmAnd#20967] [Ideas]
When recording with an external GPS receiver at 10Hz, the exported GPX file lacks fractional seconds. It would be great to have higher precision.

**Comment by sjvudp** [comment]
Maybe show example output of such GPS receiver. I guess you intend to record very fast movements, right?

**Comment by mikoreg** [comment]
I was trying to use OsmAnd to record tracks and precise timings for
Mapillary. I have a 10Hz GPS receiver which is connected through the
Bluetooth GNSS app. I know that there are many other apps like RaceChrono
which can even record in the NMEA format.

czw., 7 lis 2024, 22:45 użytkownik sjvudp ***@***.***>
napisał:

**Comment by ezhd** [comment]
ideally, app should let the user decide on time and positioning data recording precision. 
recording time in seconds precision is highly inaccurate.. not just for high frequency logging, but for any data post-processing.

**Reply by sjvudp** [reply]
> ideally, app should let the user decide on time and positioning data recording precision. recording time in seconds precision is highly inaccurate.. not just for high frequency logging, but for any data post-processing.

OTOH the size of the files will increase. Still I wonder if such devices (which is it, BTW?) do produce quality data (specifically position at rates higher than one second) to justify such a change.

**Comment by tommycrock** [comment]
On the trip recording plugin there's a logging interval below 1 second called continuous. Have you used that?

**Comment by sjvudp** [comment]
@mikoreg Reading "precission" in the title, I wanted to suggest to fix the spelling first, but then I realized that "resolution" is probably the correct word.
So maybe edit the title (I cannot).
